openedx/core/lib/teams_config.py

```diff
@@ -332,3 +360,63 @@ def _clean_max_team_size(value):
     if value < 0:
         return None
     return value
+
+
+def create_team_set_partition_with_course_id(course_id, team_sets=None):
+    """
+    Create and return the dynamic enrollment track user partition based only on course_id.
+    If it cannot be created, None is returned.
+    """
```

Comment:
This returns multiple partitions, so let's make the function name and docs true to that.

```suggestion
def create_team_set_partitions_with_course_id(course_id, team_sets=None):
    """
    Create and return the dynamic enrollment track user partitions based only on course_id.
    If they cannot be created, None is returned.
    """
```